src/cascadia/TerminalControl/TermControl.h

```diff
@@ -9,9 +9,15 @@
 #include <winrt/Microsoft.Terminal.Settings.h>
 #include "../../renderer/base/Renderer.hpp"
 #include "../../renderer/dx/DxRenderer.hpp"
+#include "../../renderer/inc/IRenderData.hpp"
 #include "../../cascadia/TerminalCore/Terminal.hpp"
 #include "../../cascadia/inc/cppwinrt_utils.h"
 
+namespace Microsoft::Console::Render
+{
+    class IRenderData;

```

Comment:
This too. In fact, Terminal.hpp includes this, so this forward decl shouldn't be neccessary